### DISCUSSION TITLE
How to set  when user register  then send email to admin ?

### DISCUSSION THREAD

**Discussion by yuhongwei380** [keycloak/keycloak#35587] [Q&A] [answered]
How to set  when user register  then send email to admin ? we need to notify admin when new user came in

**Comment by mihha** [comment, marked as answer]
I believe that this is the way

https://github.com/stfc/keycloak-email-on-user-creation

**Reply by yuhongwei380** [reply]
> I believe that this is the way
> 
> https://github.com/stfc/keycloak-email-on-user-creation

I have done for myself;  Also thanks for reply

**Reply by mihha** [reply]
Not a problem at all. I am looking for the same solution as well

How did you implement it in the end?

**Reply by yuhongwei380** [reply]
> Not a problem at all. I am looking for the same solution as well
> 
> How did you implement it in the end?

I code for myself
